# HG changeset patch
# User Rik <rik@octave.org>
# Date 1523235888 25200
#      Sun Apr 08 18:04:48 2018 -0700
# Node ID 3aed4f0ba3cd126665193a09886344fff1bf1ffd
# Parent  baadbf803b10ec8619170489be9e364d8c1f2ee4
Update some GUI strings identified as unclear by translators.

* documentation.cc, shortcut-manager.cc: Update some GUI strings identified as
unclear by translators.

diff --git a/libgui/src/documentation.cc b/libgui/src/documentation.cc
--- a/libgui/src/documentation.cc
+++ b/libgui/src/documentation.cc
@@ -83,17 +83,17 @@ namespace octave
     connect(m_help_engine, SIGNAL(setupFinished()),
             m_help_engine->searchEngine(), SLOT(indexDocumentation()));
 
     if (! m_help_engine->setupData())
       {
         QMessageBox::warning (this, tr ("Octave Documentation"),
                               tr ("Could not setup the data required for the\n"
                                   "documentation viewer. Only help texts in\n"
-                                  "the Console Widget will be available."));
+                                  "the Command Window will be available."));
         if (m_help_engine)
           delete m_help_engine;
         m_help_engine = 0;
         return;
       }
 
     // The browser
     QWidget *browser_find = new QWidget (this);
diff --git a/libgui/src/shortcut-manager.cc b/libgui/src/shortcut-manager.cc
--- a/libgui/src/shortcut-manager.cc
+++ b/libgui/src/shortcut-manager.cc
@@ -622,33 +622,33 @@ namespace octave
     // get the filename to read or write the shortcuts,
     // the default extension is .osc (octave shortcuts)
     if (action != OSC_DEFAULT)
       {
         QString file;
 
         if (action == OSC_IMPORT)
           file = QFileDialog::getOpenFileName (this,
-                                               tr ("Import shortcuts from file ..."), QString (),
+                                               tr ("Import shortcuts from file"), QString (),
                                                tr ("Octave Shortcut Files (*.osc);;All Files (*)"),
                                                nullptr, QFileDialog::DontUseNativeDialog);
         else if (action == OSC_EXPORT)
           file = QFileDialog::getSaveFileName (this,
-                                               tr ("Export shortcuts into file ..."), QString (),
+                                               tr ("Export shortcuts to file"), QString (),
                                                tr ("Octave Shortcut Files (*.osc);;All Files (*)"),
                                                nullptr, QFileDialog::DontUseNativeDialog);
 
         if (file.isEmpty ())
           return false;
 
         QSettings *osc_settings = new QSettings (file, QSettings::IniFormat);
 
         if (! osc_settings)
           {
-            qWarning () << tr ("Failed to open %1 as octave shortcut file")
+            qWarning () << tr ("Failed to open %1 as Octave shortcut file")
                         .arg (file);
             return false;
           }
         else
           {
             if (action == OSC_IMPORT)
               import_shortcuts (osc_settings);   // import (special action)
             else if (action == OSC_EXPORT)
